packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -345,6 +345,7 @@ export abstract class AbstractPureGraphManager {
   // structurally in Studio
 
   abstract HACKY_createGetAllLambda(_class: Class): RawLambda;
+  // NOTE: Input should be a regular string, can't be escaped.
```

Comment:
We need to provide a full explanation of what this function does.
Alongside a more descriptive note on what the input `assertData` should be

Reply:
@mrudula-gs @MauricioUyaguari see my comment for the comment you need to add.

Reply:
Lets remove this note as we have the comments directly on the service test